### PR TITLE
Code consistency

### PR DESCRIPTION
**Pull request by kemley76**
Fix to issue #41. 
- Switched all uses of `permissions` to `mode` in input names wherever appropriate
- Updated input names to start with `exempt` rather than ending with `exempt`
- Changed some input names to use `exemptions` instead of `exempt` ('exemptions` is at the end).
- Created a `find` command to simplify `SV-25726`

### CODE REVIEW

File: controls/SV-257926.rb [hunk 1]
```diff
@@ -31,9 +31,7 @@
   tag nist: ['CM-6 b']
   tag 'host', 'container'
 
-  crontabs = command('stat -c "%U %n" /etc/cron*').stdout.split("\n")
-
-  failing_crontabs = crontabs.reject { |c| file(c.split[1]).owned_by?('root') }
+  failing_crontabs = command('find /etc/cron* ! -user root -print0').stdout.split("\0")
```
[the remark this answers is not in the document]
Comment (wdower) [reply controls/SV-257926.rb:34]: Never mind. Crontab isn't a resource.

**Review by wdower — APPROVED** [review]

File: controls/SV-257883.rb [hunk 2]
```diff
@@ -29,14 +29,14 @@
     !virtualization.system.eql?('docker')
   }
 
-  permissions_for_libs = input('permissions_for_libs')
+  mode_for_libs = input('mode_for_libsmakethisonefail')
```
[the remark this answers is not in the document]
Comment (kemley76) [reply controls/SV-257883.rb]: It has been fixed

File: controls/SV-257890.rb [hunk 3]
```diff
@@ -32,12 +32,12 @@
     !virtualization.system.eql?('docker')
   }
 
-  exempt_home_users = input('exempt_home_users')
+  home_users_exemptions = input('home_users_exemptions')
```
[the remark this answers is not in the document]
Comment (kemley76) [reply controls/SV-257890.rb]: I have updated everything to be of the format `exempt_<thing we're exempting>`.

[the remark this answers is not in the document]
Comment (kemley76) [reply controls/SV-257890.rb]: @wdower With this, I think this PR can finally be merged in.

**Pull request merged**